mbuild/tests/test_packing.py

```diff
@@ -36,6 +36,25 @@ def test_fill_box_compound_ratio(self, h2o, ethane):
         n_water = len([c for c in filled.children if c.name == 'H2O'])
         assert n_water / n_ethane == 2
 
+    def test_fill_sphere(self, h2o):
+        filled = mb.fill_sphere(h2o, sphere=[3, 3, 3, 1.5], n_compounds=50)
+        assert filled.n_particles == 50 * 3
+        assert filled.n_bonds == 50 * 2
+
+        center = np.array([3.0, 3.0, 3.0])
+        assert np.alltrue(np.linalg.norm(filled.xyz - center, axis=1) < 1.5)
+
+    def test_fill_sphere_density(self, h2o):
+        filled = mb.fill_sphere(h2o, sphere=[3, 3, 3, 1.5], density=1000)
+        assert filled.n_particles == 921
+
```

Comment:
lets also test for a few of the things that would break this function (overdefining or underdefining the inputs, incorrect inputs, etc.)

Reply:
Essentially check that each `ValueError` listed above actually does cause a value error.